### ISSUE TITLE
Suggestion to replace UK party support graph (w10d2)

### ISSUE THREAD

**Issue by ZenoMK** [ids-s1-20/lectures#4]
Since this lecture was before a UK general election, it might be useful to replace the polling data with a more recent poll about an upcoming election, such as the US elections in November or current support for UK parties.

**Comment by mine-cetinkaya-rundel** [comment]
US elections would be good timing for this I think

**Comment by ZenoMK** [comment]
Ok. It probably would be a good idea to search for suitable graphs at a later stage since those polls get outdated so quickly. If you prefer, I could send you some suggestions until Monday.

**Comment by mine-cetinkaya-rundel** [comment]
That's true. Let's leave this issue open for now just so I know to track something down closer to time.
